frontend/src/static/js/components/webstatus-overview-content.ts

```diff
@@ -52,6 +65,23 @@ export class WebstatusOverviewContent extends LitElement {
   @property({type: Object})
   appBookmarkInfo?: AppBookmarkInfo;
 
+  @property({type: String})
+  activeQuery: string = '';
+
+  @property({type: Object})
+  savedSearch?: UserSavedSearch;
+
+  @consume({context: apiClientContext})
+  @property({attribute: false})
```

Comment:
Thanks for changing the code so that we only subscribe once and pass the data down. I like that for these properties since they are pretty much used at many levels and not sparsely. (whereas for sparely used contexts, we could selectively have components subscribe to those contexts) 🎉 

One thing to note, you could change `@property` to `@state` here.